src/fint/fint.py

```diff
@@ -260,6 +261,38 @@ def interpolate_linear_scipy(data_in, x2, y2, lon2, lat2):
     return interpolated
 
 
+def interpolate_cdo(target_grid,gridfile,original_file,output_file,variable_name,interpolation):
+    """
+    Interpolate a variable in a file using CDO (Climate Data Operators).
+
+    Args:
+        target_grid (str): Path to the target grid file.
+        gridfile (str): Path to the grid file associated with the original file.
+        original_file (str): Path to the original file containing the variable to be interpolated.
+        output_file (str): Path to the output file where the interpolated variable will be saved.
+        variable_name (str): Name of the variable to be interpolated.
+        interpolation (str): Interpolation method to be used (cdo_remapcon or cdo_remaplaf).
+
+    Returns:
+        np.ndarray: Interpolated variable data as a NumPy array.
+    """
+
+    command = [
+        "cdo",
+        "-setctomiss,0",
```

Comment:
I think this should depend on --no_mask_zero option?

Reply:
changed it